enterprise/backend/src/metabase_enterprise/data_editing/describe.clj

```diff
@@ -78,17 +77,17 @@
      :parameters (->> (for [field (sort-by sort-key fields)
                             :let [{field-values :values} field
                                   pk                     (= :type/PK (:semantic_type field))
-                                  param-mapping          (field-name->mapping (:name field))
+                                  param-map              (get param-map (keyword (:name field)))
```

Comment:
Really bad idea to pun like this here, keep the names different.

Reply:
yea I did a `/s/param-map/param-mapping` on the whole file. but yea, will change the name.